internal/envoy/v3/accesslog.go

```diff
@@ -73,3 +93,19 @@ func sv(s string) *_struct.Value {
 		},
 	}
 }
+
+func extensionConfig(extensions []string) []*envoy_config_core_v3.TypedExtensionConfig {
+	var config []*envoy_config_core_v3.TypedExtensionConfig
+
+	for _, e := range extensions {
+		// Currently only one extension exist in Envoy.
+		if e == "envoy.formatter.req_without_query" {
```

Comment:
a comment here cross-referencing `AccessLogFormatterExtensions()` in `pkg/config/parameters.go` would be helpful.